modules/ROOT/pages/troubleshoot-api-provisioning-issues.adoc

```diff
@@ -52,3 +56,54 @@ To resolve this issue:
 . Delete the API.
 . Resolve the username and password error.
 . Re-create the API.  
+
+
+[[provision-call-failed]]
+== API Provisioning Failure
+
+When you provision an API by using Anypoint Service Mesh, the asset (in Exchange) and the API instance (in API Manager) are not successfully created in Anypoint Platform.
+
+=== Causes
+
+This error can occur if you are attempting to:
+ 
+* Recreate an asset in Exchange that was more than seven days old before being deleted.
+* Provision an API with invalid values.
+
+
+=== Diagnose
+
+To diagnose this issue, run the following commands: 
+
+. `$ asmctl api list`. 
++
+The status of the API is displayed as `ProvisionCallFailed`. 
+. Run `asmctl api logs --name=<replace_with_api_name> --namespace=<replace_with_namespace>` 
++
+The following output illustratates the provisioning an API with an invalid version problem:
+
+[source,text,linenums]
+----
+DATE                   MESSAGE
+2020-05-29T17:12:36Z   Error provisioning ServiceInstance of ClusterServiceClass (K8S: "1c3b2100-5cdc-49ea-9423-dd7170e674b4" ExternalName: "anypoint-platform-api-instance") at ClusterServiceBroker "service-mesh-incluster-broker": Status: 400; ErrorMessage: Bad Request; Description: version: should match pattern "^\d+\.\d+\.\d+(-.+)?$" (ID: 49724b5d-7fad-4cc3-980e-695c2e47b383); ResponseError: <nil>
+----
+
+If no information is displayed for the previous command, run `kubectl describe serviceinstances <replace_with_api_name> -n <replace_with_namespace>`. In the `status` section of the resource, you will find the error message explaining the issue. For example: 
+
+[source,text,linenums]
+----
+Error provisioning ServiceInstance of ClusterServiceClass (K8S: "1c3b2100-5cdc-49ea-9423-dd7170e674b4"
+ExternalName: "anypoint-platform-api-instance") at ClusterServiceBroker "service-mesh-incluster-broker": Status: 410; ErrorMessage: Gone; Description: Error from Exchange: There is no asset matching given parameters. (ID: 79d0cc7f-9969-477b-816d-c822916c0290); ResponseError: <nil>' reason: ProvisionCallFailed
+----
+
+In this case, the error is caused because you are attempting to recreate an asset in Exchange that was more than seven days old before being deleted. Because Exchange does not allow that operation, you must provide a new `assetId` to create a new asset.
```

Comment:
A little doubt here. Aren't Exchange assets undeletable (through the UI) after the first 7 days?
And when deleted prior to that period, isn't that a soft delete, meaning the GAV can't be reused?

Reply:
I thought Engineering was done tech edits. This further delays the process.

Reply:
We can merge as it is @priyaayyar

Reply:
Ok thanks Tomas!